source/_integrations/klyqa.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: Klyqa
+description: Instructions on integrating Klyqa with Home Assistant.
+ha_category:
+  - Light
+  - Vacuum
+ha_release: 2022.12
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@fraizy22'
+ha_domain: klyqa
+ha_homekit: true
+ha_platforms:
+  - light
+  - vacuum
```

Comment:
Initial PRs to Home Assistant can only contain 1 platform at a time (this applies both to the documentation and the Core PRs).